packages/authhero/test/routes/universal-login/account.spec.ts

```
@@ -2,9 +2,10 @@ import { describe, it, expect } from "vitest";
 import { testClient } from "hono/testing";
 import { getTestServer } from "../../helpers/test-server";
 import { loginWithCode } from "../../helpers/login";
+import { LogTypes } from "@authhero/adapter-interfaces";
 
 describe("account", () => {
-  it("should send verification code and redirect to change-email page, then update email after code verification", async () => {
+  it.only("should send verification code and redirect to change-email page, then update email after code verification", async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)